actors/common/src/main/java/org/sunbird/learner/util/Util.java

```diff
@@ -251,6 +251,8 @@ private static void initializeDBProperty() {
     dbInfoMap.put(
         BadgingJsonKey.USER_BADGE_ASSERTION_DB,
         getDbInfoObject(KEY_SPACE_NAME, "user_badge_assertion"));
+
+    dbInfoMap.put(JsonKey.OTP, getDbInfoObject(KEY_SPACE_NAME, "otp"));
```

Comment:
Let us not add each table in dbInfoMap. Instead define below constant in corresponding DaoImpl class.

OTP_TABLE_NAME = otp